### ISSUE TITLE
long click on the title of a dictionary article

### ISSUE THREAD

**Issue by arnoulfos** [koreader/koreader#7534]
* KOReader version: 2021.03
* Device: android 7

when I click on the title of the dictionary article instead of opening a new window to search this word the dictionary window becomes blurry and nothing happen.

<details>
  <summary>Screenshots</summary>

![Screenshot_20210411-232753_KOReader](https://user-images.githubusercontent.com/72494674/114320226-a9334b00-9b1d-11eb-908f-9a801dc26a1b.png)
![Screenshot_20210411-232753_KOReader](https://user-images.githubusercontent.com/72494674/114320398-75a4f080-9b1e-11eb-99cb-b101b65a1d5f.png)

</details>

**Comment by pazos** [comment]
That's the intended behaviour. Long press on a widget makes it translucent. If you want to edit the keywork or search another word please use the edit button (placed on the top-right of your screenshot)

**Comment by arnoulfos** [comment]
When long clicking on a word in the window of the dictionary it open a search on this word but this doesn t work for the word that is the title in the dictionary article , in my screenshot εξασφαλίζω, i searched in clicking in the text εξασφάλισαν  a declined forms, the dictionary gave me the lemma εξασφαλίζω in the title, and i would like to search other dictionary entries for εξασφαλίζω , but king clicking in the word doesn t work  just this happen when the desired word is the title

**Comment by NiLuJe** [comment]
That's what the "Pen" icon is for.

**Comment by poire-z** [comment]
We removed the tap/hold on title or word because it wasn't really intuitive, hard to discover.
We replaced it with the more visible "Pen" icon ![image](https://user-images.githubusercontent.com/24273478/114422741-138fce00-9ba6-11eb-904d-2b3707e7e5fe.png).
And with this Pen icon, depending if you tap on it, or long-press on it, it will start with either the original term, or the current definition term (dunno which does what). Play with it.
